traffic_ops/app/conf/cdn.conf

```diff
@@ -34,8 +34,6 @@
         },
         "whitelisted_oauth_urls": [],
         "oauth_client_secret": "",
-        "traffic_vault_backend": "",
```

Comment:
I think we may want to keep these in here just to indicate that this is where traffic_vault is configured, but to keep them empty indicating that it's optional.